client/app/pages/queries/add-to-dashboard.html

```diff
@@ -0,0 +1,23 @@
+<div class="modal-header">
+    <button type="button" class="close" aria-label="Close" ng-click="$ctrl.close()"><span aria-hidden="true">&times;</span></button>
+    <h4 class="modal-title">Add to Dashboard</h4>
+</div>
+<div class="modal-body">
+
+    <form name="alertForm" class="form">
+      <div class="form-group">
+        <!-- <label>Limit Dashboard search to dashboards I've created? </label>
```

Comment:
Yes, there are permission checks in the Python. Also, I removed all other code that was looking for it.